### DISCUSSION TITLE
Icon for the "symbol keypad"

### DISCUSSION THREAD

**Discussion by flide** [8VIM/8VIM#132] [General]
I need a better icon for the symbol-keypad. Any ideas?

**Comment by privracki** [comment]
Why? It's already great

**Reply by flide** [reply]
Not really that great. 
![Screenshot_20201210-214418_Firefox](https://user-images.githubusercontent.com/7330631/101798224-e1575c80-3b30-11eb-897c-5aa5a46711eb.jpg)

It does not look great and symbol keypad is the only one without it's own symbol, feeling kind of left out.

**Reply by privracki** [reply]
I didn't know that's the one you were talking about. Well, I think it's fine, but perhaps the font can be made smaller, maybe just rip off Gboard and use `=\<`.
I think the best would be to give it a lighter solid background color circle

**Reply by Glitchy-Tozier** [reply]
If you're going to do something like this `=\<`, only use characters on it that _actually are on the keyboard_ though.

**Comment by Treeoctopus** [comment]
i reckon the "@" symbol is the one that most people are looking for

**Reply by Glitchy-Tozier** [reply]
I'm pretty sure the @ already exists on the X-board though  :)

Any way, I think it should be either a non-symbol-icon or a couple of symbols.

(Example for the second option:)
![Screenshot_20210310-065819_GitHub.jpg](https://user-images.githubusercontent.com/59611881/110583495-0e3a0080-816e-11eb-9ec5-02fbdaefd273.jpg)
